debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerHelper.java

```diff
@@ -705,4 +700,14 @@ public static Object getSingleResult(OracleConnection connection, String query,
             return null;
         }
     }
+
+    public static <T> T getSingleResult(OracleConnection connection, String query, Class<T> type) throws SQLException {
```

Comment:
This method and its usages should be replaced with `JdbcConnection#queryAndMap` imo.

Reply:
Done